source/index.md

```diff
@@ -372,18 +380,29 @@ Example response (snippet):
        "code": 20000
    },
    "processed_at": "2014-07-09T13:33:44+00:00",
-   "recurring": true
+   "recurring": true,
+   "_embedded": { "card": { "id": "58dabba0-e9ea-4133-8c38-bfa1028c1ed2" } }
 }
 ````
 
-Above can be repeated whenever you need to reserve money and must be followed
-by a capture transaction.
+This should (in case of non-zero amount) be followed by a capture.
+
+Subsequent authorizations are made similarly, but obviously cannot include
+neither CSC nor a PARes; either using the [cards](#cards) endpoint or by (when
+approved for CSC-less transactions) using the `card` payment method.
+
+When using the `applepay` payment method, the first recurring transaction is
+made using `applepay[*]` whereas subsequent recurring transactions are made using
+`card[]` with the details from the first transaction's `applepay[token]`.
```

Comment:
Inconsistent to use "`applepay[*]`" and `card[]`.

Also, one could use the responded card token or, when approved for CSC-less transactions, the card details in the first transaction's `applepay[token]`.